documentation/gitbook/how-to-run/multi-configs.md

```diff
@@ -32,46 +32,17 @@ you'll get a basic foward simulation of this example model. However, you might a
 flepimop simulate config_sample_2pop.yml config_sample_2pop_outcomes_part.yml
 ```
 
-if want to see what the combined configuration is, you can use the `patch` command:
+if want to create a combined configuration (e.g. to check what the combined configuration is), you can use the `patch` command:
 
 ```bash
-flepimop patch config_sample_2pop.yml config_sample_2pop_outcomes_part.yml
+flepimop patch config_sample_2pop.yml config_sample_2pop_outcomes_part.yml > config_new.yml
+cat config_new.yml
 ```
 
 You may provide an arbitrary number of separate configuration files to combine to create a complete configuration.
 
 ## Caveats
 
-At this time, only `simulate` supports multiple configuration files. Also, the patching operation is fairly crude: configuration options override previous ones completely, though with a warning. The files provided from left to right are from lowest priority (i.e. for the first file, only options specified in no other files are used) to highest priority (i.e. for the last file, its options override any other specification).
+At this time, only `simulate` supports multiple configuration files. Also, the patching operation is fairly crude: if multiple configuration files specify the same option, this will yield an error.
```

Comment:
Propose just a clarifying statement to make things a bit more explicit.

> At this time, only `simulate` directly supports multiple configuration files, and our current patching capabilities only allow for the addition of new sections as given in our tutorials. This is helpful for building models piece-by-piece from a simple compartmental forward simulation, to including outcome probabilities, and finally, adding modifier sections. If multiple configuration files specify the same higher level configuration chunks (e.g., `seir`, `outcomes`), this will yield an error.